src/Components/BookPage/BookSynopsis.js

```diff
@@ -1,65 +1,467 @@
-import React from 'react';

```

Comment:
The book synopsis file is enormous, it might be helpful to break it down into specific components! the age/grade range and additional resource cards could be separated as they have specific logic, maybe the tags could be pulled out of this file as well!